### ISSUE TITLE
[Bug] `container_id` is unconfigurable  attribute at `mongodbatlas_advanced_cluster` resource

### ISSUE THREAD

**Issue by ivan-sukhomlyn** [mongodb/terraform-provider-mongodbatlas#890]
As mentioned in the documentation for the `mongodbatlas_advanced_cluster` TF resource, it's possible to define a custom Network Peering Container ID - https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster#container_id

But the `container_id` attribute is a computed value based on source code - https://github.com/mongodb/terraform-provider-mongodbatlas/blob/d0331bc1c0c9b9d1f75cfec1a1e8b4c1ee3a88cf/mongodbatlas/resource_mongodbatlas_advanced_cluster.go#L230

### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
>> terraform version
Terraform v1.2.9
on darwin_arm64
+ provider registry.terraform.io/mongodb/mongodbatlas v1.4.6
```

### Terraform Configuration File 

```hcl
locals {
  mongo_atlas_region = upper(replace(var.region, "-", "_"))
}

resource "mongodbatlas_advanced_cluster" "product" {
  project_id = data.mongodbatlas_project.product.id

  name                   = "test-${local.env}"
  mongo_db_major_version = "6.0"
  version_release_system = "LTS"

  cluster_type = "REPLICASET"

  replication_specs {
    zone_name = "AWS-${local.mongo_atlas_region}"

    container_id = {
      "AWS:${mongodbatlas_network_container.product.region_name}" = mongodbatlas_network_container.product.container_id
    }

    region_configs {
      provider_name = "AWS"
      region_name   = local.mongo_atlas_region
      priority      = 7


      electable_specs {
        instance_size   = "M30"
        node_count      = 3
        ebs_volume_type = "STANDARD"
        disk_iops       = null
      }
...
}

resource "mongodbatlas_network_container" "product" {
  project_id       = data.mongodbatlas_project.product.id
  provider_name    = "AWS"

  atlas_cidr_block = "192.168.248.0/21"

  region_name      = local.mongo_atlas_region
}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform plan`

### Expected Behavior
Ability to define `container_id` explicitly at the `mongodbatlas_advanced_cluster` resource or fixed documentation

### Actual Behavior
```
│ Error: Value for unconfigurable attribute
│ 
│   with mongodbatlas_advanced_cluster.product,
│   on mongodb_atlas.tf line 22, in resource "mongodbatlas_advanced_cluster" "product":
│   22: resource "mongodbatlas_advanced_cluster" "product" {
│ 
│ Can't configure a value for "replication_specs.0.container_id": its value will be decided automatically based on the result of applying this configuration.
╵
```

**Comment by themantissa** [comment]
@ivan-sukhomlyn thanks for the issue.  We should just correct the documentation here.  Considering a container has to exist for a cluster to be created this isn't really a value a user needs to set, hence why compute works.  Usually users only need to think of containers if they want to do something that does not automatically create a container for them, such as setup peering first.  (Also verified it is correct in cluster so this was obviously a miss - thanks @ivan-sukhomlyn !)

**Comment by themantissa** [comment]
Internal ticket [INTMDB-463](https://jira.mongodb.org/browse/INTMDB-463)

**Comment by ivan-sukhomlyn** [comment]
@themantissa Thank you for the details and clarification!

**Comment by Zuhairahmed** [comment]
just FYI @ivan-sukhomlyn we released v1.6.1 today which includes this doc fix. feel free to re-open issue if you need anything else here. thank you.

**Comment by ivan-sukhomlyn** [comment]
Hi @Zuhairahmed 
thank you for the fix and clarification!
